wrappers/python/pyproject.toml

```diff
@@ -4,5 +4,6 @@ requires = [
     "setuptools_scm",
     "wheel",
     "cmake>=3.14",
+    "pybind11[global]>=2.10.4",
```

Comment:
* This line would force the presence of the pybind11 module (to be later found via `find_package`) right?
* What does the `[global]` do?

Reply:
- I believe it was the Windows build which couldn't find pybind11, even with this change and spawned the extra find package above
- [Docs here](https://pybind11.readthedocs.io/en/stable/installing.html#include-with-pypi). It's use here in the `pyproject.toml` build system is how it's meant to be used.  Without it, all the systems failed to locate pybind11 when build (not just windows)